pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,19 @@
+import NextAuth from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+import { PrismaAdapter } from '@next-auth/prisma-adapter';
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export const authOptions = {
+    adapter: PrismaAdapter(prisma),
+    providers: [
+        GoogleProvider({
+            clientId: process.env.GOOGLE_CLIENT_ID as string,
```

Comment:
you can also add for now these `envs` into `.env.example` so I could set up the app and test it